setup/q2rtx.sh

```diff
@@ -2,21 +2,28 @@
 
 BIN_PREFIX="."
 
+XDG_DATA_HOME := ${HOME}/.local/share
```

Comment:
This will override any user-supplied value for `XDG_DATA_HOME`. The specification says,
```
$XDG_DATA_HOME defines the base directory relative to which user-specific data files
should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to 
$HOME/.local/share should be used. 
```